test/e2e/tests/multichain/asset-list.spec.ts

```diff
@@ -75,7 +74,8 @@ describe('Multichain Asset List', function (this: Suite) {
         await loginWithoutBalanceValidation(driver);
         const assetListPage = new AssetListPage(driver);
         await switchToNetworkFromSendFlow(driver, NETWORK_NAME_MAINNET);
-        await assetListPage.checkTokenItemNumber(3);
+        // Only Ethereum network is selected so only 1 token visible
+        await assetListPage.checkTokenItemNumber(1);
```

Comment:
Cleanup test as network manager logic has changed.
We do not support selecting multiple networks now.